### PR TITLE
Document 3.6+ support in classifiers and python_requires

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

**Comment by altendky** [comment]
Relates to eerimoq/cantools#212

File: setup.py [hunk 1]
```diff
@@ -13,10 +13,15 @@
       classifiers=[
           'License :: OSI Approved :: MIT License',
           'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3.6',
+          'Programming Language :: Python :: 3.7',
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9',
```
Comment (eerimoq) [inline setup.py]: Too hard to maintain in my opinion. Please remove and I can merge the python_requires addition.

File: setup.py [hunk 2]
```diff
@@ -13,10 +13,12 @@
       classifiers=[
           'License :: OSI Approved :: MIT License',
           'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3 :: Only',
```
Comment (eerimoq) [inline setup.py]: And what when Python 4 is released? =) I suggest removing this change.

Comment (altendky) [reply setup.py]: then i'll just remove the python3 as well...  if you can't maintain a classifier for a hypothetical but unexpected python major release that has been on average every decade or so so far...

**Comment by altendky** [comment]
The way more important bit is that people installing into python 2 will be getting a py3-only version since this wasn't done prior to dropping py2 support.

**Comment by eerimoq** [comment]
Classifiers are not used in the insallation process. They are only used for PyPI search and for human to see which Python versions that are likely to be supported. python_requires is used when installing, and I think wheel names are as well.

**Comment by altendky** [comment]
Yup.

**Pull request merged**